examples/torch/classification/main.py

```diff
@@ -259,7 +259,7 @@ def validate_fn(model, epoch):
             # training function that trains the model for one epoch (full training dataset pass)
             # it is assumed that all the NNCF-related methods are properly called inside of
             # this function (like e.g. the step and epoch_step methods of the compression scheduler)
-            def train_epoch_fn(compression_ctrl, model, epoch, optimizer, **kwargs):
+            def train_epoch_fn(compression_ctrl, model, optimizer, lr_scheduler, epoch):
```

Comment:
Does this change needed in in segmentation/detection samples?

Reply:
Not critical, but better to change indeed